string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,15 @@
+/* exported isVowel */
+
+/* declare a function isVowel with one parameter character
+declare a variable x that is the character method toLowerCase
+compare the value of x to a set of 'or' logical operators of a, e, i ,o, u, y
+if true return true
+if not return false */
+
+function isVowel(character) {
+  var x = character.toLowerCase();
+  if (x === 'a' || x === 'e' || x === 'i' || x === 'o' || x === 'u' || x === 'y') {
+    return true;
+  }
+  return false;
```

Comment:
We have not covered this in class, but this can also be accomplished by using something called a `switch` statement. [Here's](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) some additional reading if you're interested!

```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```